packages/nocodb/src/plugins/s3/S3.ts

```diff
@@ -168,7 +177,10 @@ export default class S3 implements IStorageAdapterV2 {
       const data = await upload.done();
 
       if (data) {
-        return `https://${this.input.bucket}.s3.${this.input.region}.amazonaws.com/${uploadParams.Key}`;
+        let endpoint = this.input.endpoint
```

Comment:
```suggestion
        const endpoint = this.input.endpoint
```
Please use const here instead of let